uoftscrapers/scrapers/libraries/__init__.py

```diff
@@ -0,0 +1,150 @@
+from ..utils import Scraper
+from bs4 import BeautifulSoup, NavigableString, Comment
+from datetime import datetime, date
+from collections import OrderedDict
+
+
+class Libraries:
+    """A scraper for the Libraries at the University of Toronto."""
+    host = 'https://onesearch.library.utoronto.ca/'
+
+    @staticmethod
+    def scrape(location='.'):
+        Scraper.logger.info('Libraries initialized.')
+        Scraper.ensure_location(location)
+
+        for library_link in Libraries.get_library_link():
+            doc = Libraries.get_library_doc(library_link)
+            if (doc is not None):
+                Scraper.save_json(doc, location, library_link.split('/')[-1])
```

Comment:
I think all filenames should be capitalized, might be a good idea to add an `id` key with this same value for consistency.